helpers/colour-contrast/accessibility.js

```diff
@@ -0,0 +1,19 @@
+import { contrastColors, contrastRGB } from './contrast.js';
+
+const contrastMin = 4.5;
+
+// takes 2 color strings
+function isAccessible(color1, color2, myContrastMin) {
+	return contrastColors(color1, color2) >= (myContrastMin || contrastMin);
+}
+
+// takes 2 RGBColor objects
+function isAccessibleRGB(rgb1, rgb2, myContrastMin) {
+	return contrastRGB(rgb1, rgb2) >= (myContrastMin || contrastMin);
+}
+
+export {
+	isAccessible,
+	isAccessibleRGB,
+	contrastMin
+};
```

Comment:
I think I'd recommend just exporting the functions when they're defined -- that way folks can import only what they need instead of always importing everything. i.e.

```
export const contrastMin = 4.5;

export function isAccessible(...) {
}

export function isAccessibleRGB(...) {
}
```